chain/consensus/hierarchical/actors/subnet/iface.go

```diff
@@ -1,16 +1,15 @@
 package subnet
 
 import (
-	abi "github.com/filecoin-project/go-state-types/abi"
-
+	"github.com/filecoin-project/go-state-types/abi"
 	"github.com/filecoin-project/lotus/chain/consensus/hierarchical/actors/sca"
 	"github.com/filecoin-project/specs-actors/v7/actors/runtime"
 )
 
-// SubnetIface defines the minimum interface that needs to be implemented by subnet
-// actors
+// SubnetIface defines the minimum interface that needs to be implemented by subnet actors.
+// TODO: @alfonso do we need this?
```

Comment:
The idea was for this interface to be the one that users would need to implement to deploy their own subnet actors. This doesn't make much sense in the current scenario where there's no support for user-defined actors, but it'll change with FVM. 

If you feel that constantly fixing this interface with every change is making you unproductive feel free to remove it, but I'd rather keep it there for completeness until we have the FVM actors ready and we remove all this code.

Reply:
ok